Computer History Museum Archive.js

```diff
@@ -0,0 +1,1000 @@
+{
+	"translatorID": "f3b81c4e-28b4-41ae-9824-55739fe9c91a",
+	"label": "Computer History Museum Archive",
+	"creator": "Bo An",
+	"target": "^https?://www.computerhistory.org/collections/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2020-12-03 17:02:51"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	Copyright © 2020 Bo An
+	This file is part of Zotero.
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+	***** END LICENSE BLOCK *****
+*/
+
+
+
+function detectWeb(doc, url) {
+	const collection = "/collections/search/";
+	const artifact = "/collections/catalog/"
+	const isCollection = url.includes(collection)
+	const isArtifact = url.includes(artifact)
+	if (isCollection) {
+		return 'multiple';
+	}
+	else if (isArtifact) {
+	//todo it'd be better to use different items types according to item types in the collection, i.e. interview, video, audio etc, but for now 'book' has most fields needed.
+		return 'book';
+	}
+	return false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) === "multiple") {
```

Comment:
`==` is safer than `===` for strings in general.

Reply:
fixed.